includes/admin/views/add-ons.php

```diff
@@ -1,96 +1,848 @@
+<?php
+$add_ons = get_transient( 'wperp_addons' );
```

Comment:
Kindly change the key name, otherwise, this will create compatibility issues with some installation, eg: use wperp_pro_addons